style.css

```diff
@@ -98,10 +94,28 @@ background-color: white;
     z-index: 2;
     padding: 0px 15px 0px 0px;
 }
+.cart_list {
+    display: none;
+}
+.cart_list.active {
+    display: block;  
+    position: relative;
+    right: 50px;
+    z-index: 2;
+    padding: 20px 10px 10px 10px;
+    width: 100px;
+    height: 140px;
+    background-color: white;
+}
 .header_list li {
     list-style: none;
     margin: 0px 0px 0px 10px;
 }
+.cart_list li {
+    list-style: none;
+    margin: 10px 10px 10px 10px;
```

Comment:
Could be shortened

Reply:
> Could be shortened

i can reduce it to: "margin: 10px;"  (If i understand correctly)